library/common-library/templates/_secret.yaml.tpl

```diff
@@ -0,0 +1,87 @@
+{{/*
+Return local licenseKey if set, global otherwise.
+This helper is for internal use.
+*/}}
+{{- define "common.licenseKey._licenseKey" -}}
```

Comment:
Having in the signature like `common.licenseKey._licenseKey` is common?

Could we call it something like`common.licenseKey.cusomLicenceKey`?

Reply:
I would move this down, closer to the section where it is actually used

Reply:
I don't think it is common :(
Essentially I just wanted a way to very obviously tell chart writers that these helpers are internal, and not intended to be used. But there's probably a better way to do this.

What about `internal.licenseKey.licenseKey`, for example? So swapping `common` with `internal` for internal helpers.

Reply:
what I found confusing was the double "licenseKey.licenseKey". the underscore was actually fine to show that it is internal.

It would be nice to have "unexported" function with with `_`

Reply:
`common.licenseKey._actualKey` 😁

Reply:
Jokes aside, I've used the first `licenseKey` in `common.licenseKey` as a sort of "namespace" for everything related to the licenseKey, which includes the licenseKey itself. But I'm open to rename it to something else, like `common.licensing`?

Reply:
I cannot come up with a better naming, I would like to avoid having in the same object twice "key" used with two different meanings. The issue that it is licenseKey already everywhere 😢

Reply:
I don't bother if we have a `common.common.common.common` if it is readable like [this](https://github.com/newrelic/helm-charts/pull/749/files#diff-02a7e2152f1e56c1ccb8b4026261ab4d059454ac82f6f3b8cfcc099d0c294142R33) but I don't like that all the helper functions have a `common.nameOfTheTplFile.TheNameYouNeedForYourHelper` and this helper breaks this.

Reply:
I would rename this file to `license` and helpers to `common.license.licenseKey` and `common.license.licenseSecretKey` for example so we have the two `license` but not two `licenseKey`. WDYT?